x-pack/plugins/maps/public/classes/sources/mvt_single_layer_vector_source/mvt_single_layer_vector_source.tsx

```diff
@@ -98,6 +98,10 @@ export class MVTSingleLayerVectorSource
     });
   }
 
+  addFeature(geometry: Geometry | Position[]): Promise<void> {
```

Comment:
addFeature implementation not needed on MVTSingleLayerVectorSource since it extends AbstractSource and the implementation is the same as AbstractSource

Reply:
Because it implements `interface ITiledSingleLayerVectorSource` which extends `IVectorSource`, we run into type errors if we don't have this placeholder function.